### DISCUSSION TITLE
Zepeto Studio 에서 월드 만들기 안됨.

### DISCUSSION THREAD

**Discussion by jefflim69** [naverz/zepeto-studio-kor#1084] [5. Q&A]
기존 작업하던 World 삭제 후 새로 월드만들기를 시도 하는데 ,
월드 ID 등 계속 달리 해서 입력해도 "만들기" 버튼이 비활성화 상태 입니다 .

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 이용약관 준수까지 체크했는데도 만들기 버튼이 비활성화인 상태일까요?

**Reply by jefflim69** [reply]
최근 까지 UpLoad 하며 Test 이상 없이 진행하다가 , 월드를 새로 올려야 해서 삭제 후 다시 올리려니 이런 현상입니다 .
이용약관 체크가 따로 어디 있는지요 ?
![world](https://user-images.githubusercontent.com/12184282/181182078-c1e7e7ef-f03b-461a-8a36-4d672e047c2d.PNG)
?

**Reply by Zepeto-TechEvangelist** [reply]
스크롤을 조금만 아래로 내려보시면 이용약관 체크가 있고 그것까지 완료해야 만들기 버튼이 활성화 됩니다.

<img width="685" alt="스크린샷 2022-07-27 오후 3 58 30" src="https://user-images.githubusercontent.com/100118650/181182151-876bf4ef-6de8-4e92-8b97-171729bfe151.png">
